pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -222,9 +222,16 @@ func (a *ApplicationBackupController) updateWithAllNamespaces(backup *stork_api.
 		return fmt.Errorf("error updating with all namespaces for wildcard: %v", err)
 	}
 	pxNs, _ := utils.GetPortworxNamespace()
+	// Create a map to store the namespaces to be ignored for fast lookup
+	ignoreNamespaces := map[string]bool{
+		pxNs:              true,
+		"kube-system":     true,
+		"kube-public":     true,
+		"kube-node-lease": true,
```

Comment:
Are there any other openshift specific namespaces those need to be ignored?

Reply:
Right now not probably in future releases. Currently we are ignoring only the initial namespaces and portworx namespace.